### PR TITLE
Feature/unit test enhancements

### PR DESCRIPTION
**Pull request by garyfeltham**

### CODE REVIEW

File: spec/it_api_sandbox_spec.rb [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  #   end
+  #
+  #
+  #   it 'should delete the cha' do
+  #
+  #     result = api_instance.account_delete_request(cha_id)
+  #
+  #   end
+  #
+  # end
```
Comment (MichaelM5) [inline spec/it_api_sandbox_spec.rb:151]: this should not be commented

**Review by MichaelM5 — CHANGES_REQUESTED** [review]
uncomment the it_api_sandbox_spec.rb tests

**Comment by garyfeltham** [comment]
should now be uncommented

**Review by MichaelM5 — APPROVED** [review]
reviewed!

**Pull request merged**